smartsim/database/orchestrator.py

```diff
@@ -165,6 +165,19 @@ def check_cluster_status(self):  # cov-wlm
         if trials == 0:
             raise SmartSimError("Cluster setup could not be verified")
 
+    def _get_address(self):
```

Comment:
Lets make this a public method since we want users to find this. In Python, the underscore is typically used to denote methods that should only be used internal to the class.